### DISCUSSION TITLE
Building python JSBSim from source

### DISCUSSION THREAD

**Discussion by sr71684** [JSBSim-Team/jsbsim#599] [Support requests]
I am attempting to build the python jsbsim libraries from source. So far I have:

1. Cloned the latest verison of the repo
2. Followed the build instructions as listed [here](https://jsbsim-team.github.io/jsbsim-reference-manual/mypages/quickstart-building-the-program/).
```
mkdir build
cd build
cmake ..
make
```
Side note: I don't think the build flag `cmake -DINSTALL_PYTHON_MODULE=ON ..` listed in the documentation is valid, so I used this sequence of commands: 
```
cmake -DBUILD_PYTHON_MODULE=ON -DINSTALL_JSBSIM_PYTHON_MODULE=ON ..
make
make install
```
Which seems to run fine, however, when I try to import JSBSim from a python script like this
```python
import jsbsim
```
the following error is obtained:
```
Traceback (most recent call last):
  File "/home/user/dev/project/src/test_import.py", line 11, in <module>
    import jsbsim
ImportError: /home/user/anaconda3/envs/randomenv/lib/python3.9/site-packages/jsbsim.cpython-39-x86_64-linux-gnu.so: undefined symbol: _ZSt28__throw_bad_array_new_lengthv
```

Things I have tried:
- Giving the generated `jsbsim.cpython-39-x86_64-linux-gnu.so` file execute permissions with `chmod +x`. 
- Double checking I am building the jsbsim python libraries and later attempting to import them from inside the same anaconda enviornment.

Things I think might be causing the issue:
- Perhaps I'm using the wrong compiler version? 

**Environment**:
 - OS: Ubuntu 21.10
 - JSBSim version: latest
 - Language if applicable: Python, c++

**Comment by bcoconni** [comment]
This topic has been moved to JSBSim Discussions as per the issue template statement: `Issues that are neither bug reports or feature requests will be transferred to [Discussions](https://github.com/JSBSim-Team/jsbsim/discussions) by the team and the issue will be locked and closed.`.

Support for this topic will therefore continue within this discussion. Thanks for your understanding.

**Comment by bcoconni** [comment]
@sr71684, your support requests have been moved from `Issues` to `Discussions` several times now despite the explicit message that is displayed when an issue is created. Please stop doing that, thanks.
